clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -65,7 +65,7 @@ func (a *apiResponsivenessMeasurementPrometheus) Execute(config *measurement.Mea
 	var summaries []measurement.Summary
 
 	if config.PrometheusFramework == nil {
-		klog.Errorf("%s: prometheus framework is not provided!")
+		klog.Errorf("%s: prometheus framework is not provided!", a)
```

Comment:
Can we change it to warning, and make the warning message more informative for people not familiar with our test setup, i.e. let's explain what it means/what happens here by saying something like: Prometheus is disabled, skipping the measurement.

Reply:
+1 
In addition a link/information how to fix it would be appreciated.

Reply:
Changed from error to warning. Log message updated.

Reply:
It still doesn't tell or point you to any info on how to enable it. If there is no such documentation at this point, I'd add a todo to update this place once we have it.

Reply:
I just added it to README.
I'm not sure if there should explicit reference in log. The definition of this measurement is that it uses promethues server.

Reply:
Info about how to enable promethues server added to measurement log.

Reply:
IMHO this is a bit of an overkill to paste the exact flag here. 

We should only explain what happens, i.e. that the measurement will be skipped due to disabled prometheus server and that the test will proceed without it. Users can find more details either in the Measurement README.md or in the Prometheus README.md

Reply:
I think I agree with that - we shouldn't provide full help in every single error message :-)

Reply:
I changed the message so it describes only the reason of failure.
I guess we should put more focus on updating the REAME file then...

Reply:
I'm in general not a fan of keeping all documentation in readmes - people don't read this.
In general, logs, errors and code should be self-explonatory wherever possible.